spokestack/tts/clients/spokestack.py

```diff
@@ -0,0 +1,113 @@
+"""
+This module contains the Spokestack client for text to speech
+"""
+import base64
+import hashlib
+import hmac
+import json
+
+import requests
+
+
+MODES = {
+    "ssml": "synthesizeSSML",
+    "markdown": "synthesizeMarkdown",
+    "text": "synthesizeText",
+}
+
+
+class TextToSpeechClient:
+    """ Spokestack Text to Speech Client
+
+    Args:
+        key_id (str):
+        key_secret (str):
+        url (str):
+    """
+
+    def __init__(self, key_id: str, key_secret: str, url: str) -> None:
+
+        self._key_id = key_id
+        self._key = key_secret.encode("utf-8")
+        self._url = url
+
+    def synthesize_speech(
```

Comment:
The reason I went with `synthesize_speech` is because that's what polly and google speech use. There is probably a better way to do that than conform our client to their conventions.